src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -127,7 +127,7 @@ public function toolbarAction(Request $request, ?string $token = null): Response
             throw new NotFoundHttpException('The profiler must be enabled.');
         }
 
-        if (!$request->attributes->getBoolean('_stateless') && $request->hasSession()
+        if (!$request->isStateless() && $request->hasSession()
```

Comment:
Before it was true|false only
Now it may be null as well
Is it desired?

Reply:
Read https://github.com/symfony/symfony/pull/57852#issuecomment-2255360719, but it's reverted in https://github.com/symfony/symfony/pull/57852/commits/a17bfcd0eb0e33d29eb41da5bac40fae9b3b0b67 as wanted by stof.

Reply:
Good 👍🏻